src/components/Albums/AlbumCard/AlbumCard.scss

```diff
@@ -0,0 +1,21 @@
+.album-card {
+  margin: 0 0.9rem 2.5rem 0.9rem;
```

Comment:
the component should not know about its positioning. Please move this margin top parent component

Reply:
Corrected